onboarding/scaffolding.py

```diff
@@ -75,6 +76,10 @@ class OnboardingScaffolding(pydantic.BaseModel):
     # on the server-side.
     address_verified: bool = False
 
+    lease_type: str = ""
```

Comment:
how do you feel about changing this to `housing_type` (either here or letting me do it in a future PR), since we're editing it anyway?

Reply:
Oh sure.  I wonder if it makes sense to do it in a separate PR so that we can change it everywhere (including in `session.onboardingInfo`) and not just here?

Reply:
Otherwise my concern is that if we don't call it "housing type" across the whole codebase, then future contributors may easily get confused and wonder if housing type is different from lease type...

Reply:
yeah that's fair. i'll do a follow up PR to mine that changes it everywhere reasonable (without requiring migrations or v2s of mutations)